client/src/modules/dashboard/Events/components/EventForm.tsx

```diff
@@ -260,7 +265,7 @@ const EventForm: React.FC<EventFormProps> = (props) => {
           Invite only
         </Checkbox>
 
-        <FormControl isRequired>
+        <FormControl marginBlock={'1em'}>
```

Comment:
```suggestion
        <FormControl isRequired marginBlock={'1em'}>
```
I know it doesn't make much difference, since one is always selected, but since this is required, we should keep that explicit.